f8a_worker/workers/CVEchecker.py

```diff
@@ -19,16 +21,46 @@ class CVEcheckerTask(BaseTask):
     schema_ref = SchemaRef(_analysis_name, '3-0-0')
 
     @staticmethod
+    def get_cve_impact(id_):
+        score = 0
+        vector = ''
+        severity = ''
+        if id_:
+            url = "https://nvd.nist.gov/vuln/detail/{id_}".format(id_=id_)
+            response = requests.get(url)
+            if response.status_code == 200:
+                score_v2, score_v3 = 0, 0
+                severity_v2, severity_v3, vector_v2, vector_v3 = '', '', '', ''
```

Comment:
would not this be nicer? 

`severity_v2 = severity_v3 = vector_v2 = vector_v3 = ''`